Watchman.Web/Areas/Administration/Models/ConfigurationItemDto.cs

```diff
@@ -1,7 +1,9 @@
-﻿using Watchman.DomainModel.Configuration;
+﻿using TypeGen.Core.TypeAnnotations;
+using Watchman.DomainModel.Configuration;
 
 namespace Watchman.Web.Areas.Administration.Models
 {
+    [ExportTsClass(OutputDir = "ClientApp/src/models")]
```

Comment:
może dałoby sie to refleksją jakoś ogarnąć
bo ręczne ustawianie dla każdego pliku może być troche męczące
no chyba że sie nie da
w TypeLite chyba sie nie dało

Reply:
Można to zrobić tak jak wyżej za pomocą atrybutów lub stworzyć taką klasę : 

`public class MyGenerationSpec : GenerationSpec`
`{`
   ` public MyGenerationSpec()`
    `{`
        `AddClass <MyClassDto>();`
        `...`
   ` }`
`}`
i w niej wpisać wszystkie klasy.

Reply:
albo można zrobić to w taki sposób jak napisałeś
ale refleksją zebrać wszystkie typy które są z Watchman.Web.Areas i które w nazwie klasy zawierają "Dto" 

to nie musi być super wydajne bo to poboczny i proces i używany jedynie podczas budowania aplikacji